setup.py

```diff
@@ -145,8 +145,12 @@
     # - sonic-config-engine
     # - swsssdk
     # - tabulate
+    # - click
     install_requires=[
+        'click-default-group',
```

Comment:
Why add this? I removed this dependency recently.

Reply:
yes, I don't need it. Removed it.